src/Commands/InstallCommand.php

```diff
@@ -5,16 +5,18 @@
 use Illuminate\Console\Command;
 use Illuminate\Contracts\Filesystem\FileNotFoundException;
 use Illuminate\Filesystem\Filesystem;
-use Illuminate\Support\Arr;
-use Illuminate\Support\Str;
+// use Illuminate\Support\Str;
 use KoalaFacade\DiamondConsole\Actions\Composer\ResolveComposerAutoLoaderAction;
+use KoalaFacade\DiamondConsole\Commands\Concerns\HasArguments;
 use KoalaFacade\DiamondConsole\Enums\Layer;
 use KoalaFacade\DiamondConsole\Support\Source;
 use Throwable;
 
 class InstallCommand extends Command
 {
-    protected $signature = 'diamond:install {--skip-refactor}';
+    use HasArguments;
+
+    protected $signature = 'domain:install {domain} {--skip-refactor}';
```

Comment:
remove --skip-refactor since we disabled it

Reply:
Oh yeah, I forgot this